examples/src/bin/multi_windows.rs

```diff
@@ -1,5 +1,4 @@
-use gio::prelude::*;
-use glib::clone;
+use gtk::glib;
```

Comment:
Same here too... :'(

Reply:
All of those are unrelated to my changes though

Reply:
Yup yup, don't worry. It was mostly for me, sorry for the noise.

Reply:
aha i misunderstood :p I thought I had to fix those as well hehe